proto/src/determined/task/v1/task.proto

```diff
@@ -89,6 +98,8 @@ message Task {
   google.protobuf.Timestamp start_time = 5;
   // End timestamp if completed.
   optional google.protobuf.Timestamp end_time = 6;
+  // The configuration of the task.
+  google.protobuf.Struct config = 7;
```

Comment:
returning experiment config in the list experiment was a giant mistake that we still need to undo

These configs can potentially be massive megabyte size yaml files

Just echoing this concern we don't do the same for ```getTasks```

and this being required implies that it would need to return it?

Reply:
made optional. Only returned when used for get config